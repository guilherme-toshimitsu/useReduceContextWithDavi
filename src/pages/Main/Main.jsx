import React from 'react';
import Routes from '@components/Routes';
import routes from '@routes';
import { BrowserRouter } from 'react-router-dom';

// import Routes
const Main = () => {
  return (
    <BrowserRouter>
      <Routes routes={routes} />
    </BrowserRouter>
  );
};

export default Main;
