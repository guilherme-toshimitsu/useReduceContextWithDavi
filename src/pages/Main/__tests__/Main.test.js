import React from 'react';
import { render } from '@testing-library/react';
import Main from '../';

describe('<Main> Main Container Page', () => {
  it('renders', () => {
    const container = render(<Main />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
