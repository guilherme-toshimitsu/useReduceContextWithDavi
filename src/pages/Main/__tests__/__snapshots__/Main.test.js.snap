// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Main> Main Container Page renders 1`] = `undefined`;
